test/unit/knex.js

```diff
@@ -171,17 +238,18 @@ describe('knex', () => {
   });
 
   it('throws if client module has not been installed', () => {
-    expect(Knex({ client: 'oracle' })).to.throw(
-      /Knex: run\n$ npm install oracle/
+    expect(() => {
+      Knex({ client: 'oracledb', connection: {} });
+    }).to.throw(
+      "Knex: run\n$ npm install oracledb --save\nCannot find module 'oracledb'"
     );
   });
 
   describe('async stack traces', () => {
     it('should capture stack trace on query builder instantiation', () => {
-      const knex = Knex({
-        ...sqliteConfig,
-        asyncStackTraces: true,
-      });
+      const knex = Knex(
+        Object.assign({}, sqliteConfig, { asyncStackTraces: true })
```

Comment:
Was there a problem with original implementation?

Reply:
Yes, Node 6 doesn't support spread, and we don't transpile tests. It used to work because it was never run in CI :)